code-challenges.test.js

```diff
@@ -1,38 +1,75 @@
+// // Pseudo coding is REQUIRED
+// // If you get stuck, please leave comments to help us understand your thought process
 
-// Reminder: The test will call your function
-// Run the file with the following command: $ yarn jest
+// // Use test driven development to complete the following questions
+// // Add appropriate dependencies: $ yarn add jest
 
-// --------------------1) Create a function that takes in an array, removes the first item from the array and shuffles the remaining items.
+// // Reminder: The test will call your function
+// // Run the file with the following command: $ yarn jest
 
-// a) Create a test with an expect statement using the variable provided.
-// HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
+// // --------------------1) Create a function that takes in an array, removes the first item from the array and shuffles the remaining items.
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+// // a) Create a test with an expect statement using the variable provided.
+// // HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
+// describe ("shuffleRemovingFirst", () => {
+//     it("Create a function that takes in an array, removes the first item from the array and shuffles the remaining items", () => {
+//         const colorsArr1 = shuffleRemovingFirst([...colors1]);
+//         const ColorsArr2 = shuffleRemovingFirst([...colors2]);
+//         expect (colorsArr1).toEqual(expect.arrayContaining(colors1.slice(1)));
+//         expect (ColorsArr2).toEqual(expect.arrayContaining(colors2.slice(1)));
+//     });
+// });
 
-// b) Create the function that makes the test pass.
+// const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+// // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
+// const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+// // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+
+// // b) Create the function that makes the test pass.
+// const shuffleRemovingFirst = (arr) => {
+//     const RemovedElement = arr.shift();
+//     arr.sort(() => Math.random() - 0.5);
+//     return arr;
+// }
 
 // Pseudo code:
+// input: an array
+// output: a new array with the first element removed and remaining elements shuffled
+// create functin name ===> shuffleRemovingFirst
+// the function is taking in an array so arr would be a parameter.
+// create logic that will first remove first element from array.
+// create logic that will shuffle remaining elements in array
```

Comment:
Let's expand on pseudocode a bit more in future assessments.  For instance, what are the built-in methods you are utilizing to shuffle the remaining events and what is it's purpose.